spk/emby/src/dsm-control.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+# Package
+PACKAGE="emby"
+DNAME="Emby"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+USER="${PACKAGE}"
+
+MONO_PATH="/usr/local/mono/bin"
+MONO="${MONO_PATH}/mono"
+EXE_FILE="${INSTALL_DIR}/share/emby/MediaBrowser.Server.Mono.exe"
+
+EXTRA_LIBS="/usr/local/mediainfo/lib:/usr/local/imagemagick/lib:${INSTALL_DIR}/lib"
+FFMPEG="/usr/local/ffmpeg/bin"
+
+COMMAND="env PATH=${MONO_PATH}:${PATH} LD_LIBRARY_PATH=${EXTRA_LIBS} ${MONO} -- ${EXE_FILE} -programdata ${INSTALL_DIR}/var -ffmpeg ${FFMPEG}/ffmpeg -ffprobe ${FFMPEG}/ffprobe"
+
+start_daemon ()
+{
+    start-stop-daemon -c ${USER} -S -q -b -m -p ${INSTALL_DIR}/var/emby.pid -N 10 -x ${COMMAND} > /dev/null
```

Comment:
Fwiw, I wouldn't mind if you set `PID_FILE = ${INSTALL_DIR}/var/emby.pid` on L15 or thereabouts, and did ${PID_FILE} here. That's the usual approach.